llmfoundry/models/utils/__init__.py

```diff
@@ -6,12 +6,17 @@
 from llmfoundry.models.utils.hf_prefixlm_converter import (
     add_bidirectional_mask_if_missing, convert_hf_causal_lm_to_prefix_lm)
 from llmfoundry.models.utils.meta_init_context import init_empty_weights
+from llmfoundry.models.utils.misc import is_torch_2_or_higher
 from llmfoundry.models.utils.param_init_fns import (  # type: ignore
     MODEL_INIT_REGISTRY, generic_param_init_fn_)
 
 __all__ = [
     'AutoTokenizerForMOD', 'adapt_tokenizer_for_denoising',
     'convert_hf_causal_lm_to_prefix_lm', 'init_empty_weights',
     'add_bidirectional_mask_if_missing', 'generic_param_init_fn_',
-    'MODEL_INIT_REGISTRY'
+    'MODEL_INIT_REGISTRY', 'is_torch_2_or_higher'
 ]
+
+if is_torch_2_or_higher():
+    from llmfoundry.models.utils.tensor_parallel import PairwiseSequenceParallel
```

Comment:
I guess this one doesn't need to be in a try/except for some reason? thats cool lol